ui/app/helpers/utils/i18n-helper.js

```diff
@@ -39,13 +40,41 @@ export const getMessage = (localeCode, localeMessages, key, substitutions) => {
   }
   const entry = localeMessages[key]
   let phrase = entry.message
+
+  const requiresWrappers = phrase.match(/\{\{.+?\}\}/)
+
+  const hasSubstitutions = Boolean(substitutions && substitutions.length)
+  const hasReactSubstitutions = hasSubstitutions && substitutions.some((element) => typeof element === 'function' || typeof element === 'object')
```

Comment:
Nit: could you add a newline here somewhere? I had to scroll horizontally even when looking at this fullscreen.

Reply:
Done in 247b12834